src/index.css

```diff
@@ -0,0 +1,152 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fafafa;
+}
+
+.header {
+  width: 100%;
+  padding: 20px 100px;
+  border-bottom: 1px solid rgb(197, 197, 197);
+  background-color: #fff;
+  display: flex;
+}
+
+.header h1 {
+  color: #0290ff;
+}
+
+.header ul {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 30px;
+  padding: 10px 20px;
+  width: 180px;
+  list-style-type: none;
+}
+
+.links {
+  color: #5e5e5e;
+  text-decoration: none;
+}
+
+.header ul li:hover {
+  cursor: pointer;
+}
+
+.header img {
+  width: 30px;
+  height: 30px;
+  margin-left: 650px;
+}
+
+.books-list {
+  width: 100%;
+  padding: 20px 100px;
+  background-color: #fafafa;
+}
+
+.singlBook {
+  display: flex;
+  justify-content: space-between;
+  margin: 15px 0;
+  padding: 20px;
+  background-color: #fff;
+  border-radius: 10px;
+}
```

Comment:
- [ ] As requested by the previous reviewer, kindly ensure your design is as close to the template as possible.
Take a look at the way the book title and author are arranged, also, the `remove`, `edit`, and `comment` button below the names are missing. Kindly fix it.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/187050286-7369447d-84cf-4f6c-a3bb-ee2212b8e471.PNG)
Zemplin template
![187049246-d189ea50-be0b-4658-8330-74e2355e82d4](https://user-images.githubusercontent.com/85095702/187050294-1befe17d-70e9-4b08-aba5-a5be0aa62123.png)